scipy/signal/fir_filter_design.py

```diff
@@ -594,3 +596,106 @@ def remez(numtaps, bands, desired, weight=None, Hz=1, type='bandpass',
     bands = np.asarray(bands).copy()
     return sigtools._remez(numtaps, bands, desired, weight, tnum, Hz,
                            maxiter, grid_density)
+
+
+def firls(numtaps, bands, desired, weight=None, fs=2):
```

Comment:
I chose this one's to match what MATLAB and Octave use. Better to be
consistent(ish) ourselves or have least surprise for people changing over?
It's about 50/50 for me but I think I agree internal consistency (change to
1.) is better.